internal/cli/cmd/organization/organization.go

```diff
@@ -0,0 +1,271 @@
+package organization
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"github.com/apecloud/kubeblocks/internal/cli/printer"
+)
+
+const (
+	APIURL  = "http://a8ff89cbeec444c82b90c5f83a117b39-16361bbd933bde33.elb.cn-northwest-1.amazonaws.com.cn:8086"
+	APIPath = "api/v1"
+)
+
+type Organizations struct {
+	Items []OrgItem `json:"items"`
+}
+
+type OrgItem struct {
+	ID          string `json:"id"`
+	Name        string `json:"name"`
+	Role        string `json:"role"`
+	Description string `json:"description"`
+	DisplayName string `json:"displayName"`
+	CreatedAt   string `json:"createdAt"`
+	UpdatedAt   string `json:"updatedAt"`
+}
+
+type CurrentOrgAndContext struct {
+	CurrentOrganization string `json:"currentOrganization"`
+	CurrentContext      string `json:"currentContext"`
+}
+
+type Organization interface {
+	getOrganization(token string, name string) (*OrgItem, error)
+	GetOrganizations(token string) (*Organizations, error)
+	switchOrganization(token string, name string) (string, error)
+	addOrganization(token string, body []byte) error
+	deleteOrganization(token string, name string) error
+	IsValidOrganization(token string, name string) (bool, error)
+}
+
+type OrganizationOption struct {
+	Name         string
+	OutputFormat string
+
+	Organization Organization
+
+	genericclioptions.IOStreams
+}
+
+func newOrganizationOption(streams genericclioptions.IOStreams) *OrganizationOption {
+	return &OrganizationOption{
+		IOStreams: streams,
+		Organization: &CloudOrganization{
+			APIURL:  APIURL,
+			APIPath: APIPath,
+		},
+	}
+}
+
+func NewOrganizationCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "org",
+		Short: "Organization command.",
+	}
+	cmd.AddCommand(
+		newOrgListCmd(streams),
+		newOrgSwitchCmd(streams),
+		newOrgCurrentCmd(streams),
+		newOrgDescribeCmd(streams),
+	)
+
+	return cmd
+}
+
+func newOrgListCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	o := newOrganizationOption(streams)
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List organizations.",
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(o.complete(args))
+			cmdutil.CheckErr(o.validate(cmd))
+			cmdutil.CheckErr(o.runList())
+		},
+	}
+
+	return cmd
+}
+
+func newOrgSwitchCmd(streams genericclioptions.IOStreams) *cobra.Command {
+	o := newOrganizationOption(streams)
+
+	cmd := &cobra.Command{
+		Use:   "switch",
+		Short: "Switch organization.",
```

Comment:
Currently, some of the commands in kbcli directly correspond to a certain Kubernetes cluster, while others are specific to the Cloud. It is not evident from the usage or description of a newly added command whether it is only applicable to the Cloud or not.

Reply:
ACK